services/skus/datastore_test.go

```diff
@@ -498,7 +502,7 @@ func createOrderAndIssuer(t *testing.T, ctx context.Context, storage Datastore,
 
 	repo := repository.NewIssuer()
 	issuer, err := repo.Create(ctx, storage.RawDB(), model.IssuerNew{
-		MerchantID: test.RandomString(),
+		MerchantID: model.MerchID,
```

Comment:
There is only one merchant, and that is us, `brave.com`. This randomisation was totally useless.